config/model_configs/sphere_baroclinic_wave_rhoe_hughes2023.yml

```diff
@@ -0,0 +1,18 @@
+dt_save_state_to_disk: "2days"
+initial_condition: "DryHughes2023BaroclinicWave"
+topography: "Hughes2023"
+topo_smoothing: false
+implicit_diffusion: true
+approximate_linear_solve_iters: 2
+moist: "equil"
+dt: "200secs"
+t_end: "10days"
+ode_algo: SSPKnoth
```

Comment:
```suggestion
```

Reply:
(This is the default)

Reply:
Ah yes, I've updated the help string in the default config.